packages/cli/src/commands/start.test.ts

```diff
@@ -27,10 +30,20 @@ function generateRandomAlphanumericString(length: number): string {
   return result;
 }
 
-const PACKAGE_MANAGERS_WITHOUT_YARN = PACKAGE_MANAGERS.filter(packageManager => packageManager !== "yarn");
+const execAsyncMock = vi.spyOn(Utils, "execAsync");
+
+afterEach(() => {
+  execAsyncMock.mockClear();
+});
+
+const TEST_PACKAGE_MANAGERS = (process.env.START_COMMAND_TEST_PACKAGE_MANAGERS?.split(",") ?? PACKAGE_MANAGERS) as PackageManager[];
+if (TEST_PACKAGE_MANAGERS.length === 0 || !PACKAGE_MANAGERS.includes(TEST_PACKAGE_MANAGERS.at(0) as PackageManager)) {
+  throw new Error("Invalid package manager");
+}
+
 describe("start command integration test", () => {
   it.todo("we support yarn but it doesn't work on CI, so we need to fix it");
-  describe.each(PACKAGE_MANAGERS_WITHOUT_YARN)("packageManager: %s", { timeout: 1_000_000, concurrent: true }, (packageManager) => {
```

Comment:
remove `concurrent` because mock function doesn't work in concurrent environment